rpa-home/modules/ROOT/pages/release-notes/rpa-builder-6.1-release-notes.adoc

```diff
@@ -7,7 +7,14 @@ RPA Builder is the on-premise application that enables you to build automations
 // == What's New
 include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
 
-* The new Mail Session (Outlook with OAuth) Action Step enables you to establish a connection to a mail server using MS Outlook and authenticate using OAuth to perform mail operations as reading, sending, and changing emails. 
+* The new Mail Session (Outlook with OAuth) Action Step enables you to establish a connection to a mail server using MS Outlook and authenticate using OAuth to perform mail operations such as reading, sending, and changing emails.
+* The new Credentials for OAuth Action Step enables you to configure OAuth credentials once and reuse them within other Action Steps.
```

Comment:
I would emphasize here that the OAuth credentials can be generated with the help of this action step at workflow runtime, the "Once" bothers me here and something with "at runtime" is missing